api/grpcservice.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
```

Comment:
Nit: I think `api/grpc_service.proto` will be more readable.

Reply:
@wora  how about api/grpc_cluster.proto if we don't like api/grpc_service.proto?
The message defined here is used to reference a cluster that is running a grpc_service so I think grpc_service|cluster.proto would be best.
wdyt?
cc @htuch 
I'll address the other feedback and do another push.

Reply:
api/grpc_service.proto is fine. Thanks.